python-package/lightgbm/sklearn.py

```diff
@@ -984,6 +984,6 @@ def fit(self, X, y,
     _base_doc = fit.__doc__
     _before_early_stop, _early_stop, _after_early_stop = _base_doc.partition('early_stopping_rounds :')
     fit.__doc__ = (_before_early_stop
-                   + 'eval_at : list of int, optional (default=[1, 2, 3, 4, 5])\n'
+                   + 'eval_at : iterable of int, optional (default=[1, 2, 3, 4, 5])\n'
```

Comment:
Or we can silently change default from list to tuple and continue asking users to provide only lists.
```suggestion
                   + 'eval_at : iterable of int, optional (default=(1, 2, 3, 4, 5))\n'
```